### PR TITLE
parallel operators 

### PR DESCRIPTION
**Pull request by suwilanjichisanga**
I added Parallel Operators to ivy/functional/frontends/jax/lax/linalg.py

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ZoeCD** [comment]
Hi! There are some branch conflicts. Could you please update your fork to resolve these conflicts? Also, each PR should be linked to an issue (or multiple issues) according to the functions being worked on. You can read more about the steps to contribute to Ivy [here](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis) 🙂

**Comment by suwilanjichisanga** [comment]
@ZoeCD can you only look at Parallel Operators to ivy/functional/frontends/jax/lax/linalg.py, an issue has been added. The other changes were my first attempt which were not linked to an issue. If you may review Parallel Operators to ivy/functional/frontends/jax/lax/linalg.py

**Comment by ZoeCD** [comment]
I saw the closed issue. You need to create an issue per function worked on. For example, one titled `all-to-all` and another one titled `psum`. Then link those issues to this PR so they can be automatically closed once the PR is resolved. Also, each function needs to have a test function. If you prefer, [here](https://www.youtube.com/playlist?list=PLwNuX3xB_tv8Gf2sELkXSqCCWcUq-AbrG) is our contribution youtube series that explains the steps to contribute to the codebase. Let me know if you have any questions! 🙂

**Comment by suwilanjichisanga** [comment]
Hi @ZoeCD I created the issues, but I am unable to access development. I want to link the pull request to the issues.

**Comment by ZoeCD** [comment]
Please comment the link to the issues, and I'll add them in the development section

**Comment by suwilanjichisanga** [comment]
I already commented the links @ZoeCD

**Comment by suwilanjichisanga** [comment]
alright this is the link to the issues https://github.com/unifyai/ivy/issues/9246

**Comment by ZoeCD** [comment]
Issues linked. Please add the corresponding tests and fix the merge conflicts. Let me know if you have any questions 🙂
Happy new year!

**Comment by suwilanjichisanga** [comment]
@ZoeCD how can I add the corresponding tests?

File: ivy/functional/frontends/tensorflow/math.py [hunk 1]
```diff
@@ -16,30 +16,33 @@ def accumulate_n(inputs, input_type=None, shape=None, dtype=None, name=None):
...
 @to_ivy_arrays_and_back
-def add(x, y, name=None):
+def add(x, y):
     x, y = promote_types_of_tensorflow_inputs(x, y)
     return ivy.add(x, y)
 
 
 @to_ivy_arrays_and_back
+
+def argmax(input, axis, output_type=None):
```
Comment (ZoeCD) [inline ivy/functional/frontends/tensorflow/math.py:26]: I think this is a merge mistake that went unnoticed

File: ivy/functional/frontends/tensorflow/math.py [hunk 2]
```diff
@@ -16,30 +16,33 @@ def accumulate_n(inputs, input_type=None, shape=None, dtype=None, name=None):
...
+
     if output_type in ["uint16", "int16", "int32", "int64"]:
         return ivy.astype(ivy.argmax(input, axis=axis), output_type)
     else:
         return ivy.astype(ivy.argmax(input, axis=axis), "int64")
 
 
 @to_ivy_arrays_and_back
-def asinh(x, name="asinh"):
+def asinh(x):
```
Comment (ZoeCD) [inline ivy/functional/frontends/tensorflow/math.py:38]: Question. Were all these formatting changes made by the lint pre-commit?

File: ivy/functional/frontends/tensorflow/math.py [hunk 3]
```diff
@@ -359,16 +362,20 @@ def zero_fraction(value, name="zero_fraction"):
 
 
 @to_ivy_arrays_and_back
+
+def argmin(input, axis=None, output_type="int64"):
```
Comment (ZoeCD) [inline ivy/functional/frontends/tensorflow/math.py:366]: Another merge error

File: ivy/functional/frontends/jax/lax/linalg.py [hunk 4]
```diff
@@ -38,3 +38,34 @@ def symmetrize(x):
         x = symmetrize(x)
 
     return ivy.eigh(x, UPLO=UPLO)
+
+  @to_ivy_arrays_and_back
+    def all_gather(x, axis_name, *, axis_index_groups=None, axis=0, tiled=False):
```
Comment (ZoeCD) [inline ivy/functional/frontends/jax/lax/linalg.py:43]: All functions must to have a test. You can see plenty of examples under the folder `ivy_tests`. As you can see there, all our tests are organized by categories. For example, for the jax linalg functions, the tests must be in this file [ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_linalg.py](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_linalg.py)

Comment (ZoeCD) [reply ivy/functional/frontends/jax/lax/linalg.py:43]: I recommend you to read our [contribution guide](https://lets-unify.ai/ivy/contributing/open_tasks.html#frontend-apis). You will find step-by-step instructions on how to contribute to the codebase. You can also find function and tests examples 🙂

**Review by ZoeCD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**